client/src/components/main/coffee_shop/CoffeeShopIndex.scss

```diff
@@ -1,52 +1,31 @@
 .panel-coffee-subtitle {
   font-weight: 600;
-  font-size: 13px;
   color: #515151;
   margin-bottom: 10px;
 }
 
 .panel-coffee {
-  font-size: 14px;
   margin-top: 15px;
 }
 
-.panel-processing {
-  svg {
-    fill: brown;
```

Comment:
Were you thinking that the colors don't look great on the panels?

Also, I originally added the margin-right b/c they were scrunching together on certain screen sizes. Did you see that this problem wasn't happening on your end?

Reply:
I moved the color here into the icon itself.
Here is where the color prop is being passed in:
https://github.com/hegelocampus/Roastia/blob/152b2f4a1ae2feea3a62725850d288bd2aa4bb6b/client/src/components/util/icons.js#L37
And here is where the color is being declared in the style-sheet:
https://github.com/hegelocampus/Roastia/blob/152b2f4a1ae2feea3a62725850d288bd2aa4bb6b/client/src/components/util/icons.js#L5